src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -493,4 +493,19 @@ public static String convertToEmptyStringIfNull(String str) {
         return (str == null) ? "" : str;
     }
 
-}
+    /**
+     * Trims head of the String if it is longer than specified Length.
+     *  E.g., String "12345678" with maximumStringLength = 6, returns "345678"
+     * @param String
+     * @param maximumLength - required length of the string
```

Comment:
1)@param inputString
   @param maximumStringLength

2)@param inputString - (Description) 
   @param maximumStringLength - (Description) 

How am  I supposed to do ? 1 or 2 ?